lib/ProductOpener/Display.pm

```diff
@@ -6851,6 +6446,7 @@ HTML
 	# init javascript code
 
 	$html =~ s/<initjs>/$initjs/;
```

Comment:
`$html` is empty at this point, so that all of the following fix-ups don't do anything. IMHO: Move https://github.com/openfoodfacts/openfoodfacts-server/blob/8ad091812d7e88d2e52c8e8e0def72e4f2609a6c/lib/ProductOpener/Display.pm#L6208 to https://github.com/openfoodfacts/openfoodfacts-server/blob/8ad091812d7e88d2e52c8e8e0def72e4f2609a6c/lib/ProductOpener/Display.pm#L6479 and just use `$html` instead of `$temp_html`, and then run https://github.com/openfoodfacts/openfoodfacts-server/blob/8ad091812d7e88d2e52c8e8e0def72e4f2609a6c/lib/ProductOpener/Display.pm#L6419-L6448 after `$tt->process`. This should fix issues like not all images being requested from the `static` subdomain.